libs/VyfakturujAPI.class.php

```diff
@@ -509,4 +509,28 @@ protected function fetchDelete($path, $data = null)
         return $this->fetchRequest($path, self::HTTP_METHOD_DELETE, $data);
     }
 
+    /**
+     * @return string
+     */
+    public function getEndpointUrl()
+    {
+        return $this->endpointUrl;
+    }
+
+    /**
+     * @param string $endpointUrl
+     * @throws VyfakturujAPIException
+     */
+    public function setEndpointUrl($endpointUrl)
+    {
+        if(preg_match('~https?://~i', $endpointUrl) !== 1) {
+            throw new VyfakturujAPIException(sprintf(
+                'Endpoint URL must be a valid absolute URL on HTTP(S) protocol, \'%s\' is not valid absolute URL',
```

Comment:
Je to jenom o stylu: nebylo by lip využit " a ne '?

`"Endpoint URL must be a valid absolute URL on HTTP(S) protocol, '%s' is not valid absolute URL"`

Je to lip čitelně, na můj pohled

Reply:
Rozumím, je to celkem správná připomínka. Nicméně když si v PhpStormu zapneš [Extended Inpections)(https://plugins.jetbrains.com/plugin/7622-php-inspections-ea-extended-/), které do kódu vkládají bezpečnostní upozornění na možné problémy, tak zjistíš, že v situaci, kdy není `"ve stringu použitá $proměnná"`, tak se doporučuje používat striktně `'`apostrofy, aby se zabránilo nechtěnému vložení proměnné a tím možnému XSS.